defender/downloaded/Contract_cToken.js

```diff
@@ -3,9 +3,24 @@ const axios = require('axios');
 const axiosRetry = require('axios-retry');
 const ethers = require('ethers');
 
+// function to calculate the delay until the next request attempt
+// returns a value specified in milliseconds
+function retryDelayFunc(retryCount) {
+  // 300 seconds total in Autotask execution to perform retries
+  // #   time   attempt
+  // 0 -   0s - initial request
```

Comment:
It probably doesn't matter too much, but 2^0 power is 1.  So if 0 is the first retryCount, then the delay won't be 0, it'll be 20s

```
> (2**0)*20*1000;
20000
> (2**1)*20*1000;
40000
```

Reply:
I was fairly certain that it started at `1`, but I made a script to double check. TL;DR: It starts at `1`.

package.json
```json
{
  "name": "tmp",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "axios": "0.27.2",
    "axios-retry": "^3.3.1",
    "express": "^4.18.2"
  }
}
```

index.js
```javascript
const axios = require('axios');
const axiosRetry = require('axios-retry');
const express = require('express');

axiosRetry(axios, {
  retries: 3,
  retryDelay: delay,
});

function delay(retryCount) {
  console.log(retryCount);
  return retryCount*1000;
}

const app = express();

app.get('/500', (req, res) => {
  res.status(500).send('Internal Server Error');
});

app.listen(3000, () => {
  console.log('Server is running on port 3000');
});

async function main() {
  const url = 'http://localhost:3000/500';
  const response = await axios.get(url);
}

main()
  .then(() => process.exit(0))
  .catch((err) => {
    console.error('Error');
    process.exit(1);
  });
```

Output:
```shell
Server is running on port 3000
1
2
3
Error
```

This took a while because the `axios` maintainers apparently broke something when they released `v1.0.0` and later: https://github.com/axios/axios/issues/5089